src/ds5/ds5-device.cpp

```diff
@@ -1314,10 +1314,14 @@ namespace librealsense
                                        make_attribute_parser(&md_mipi_depth_mode::input_height,
                                                              md_mipi_depth_control_attributes::input_height_attribute,
                                                              md_prop_offset));
-        depth_sensor.register_metadata(RS2_FRAME_METADATA_SUB_PRESET_INFO, // added for mipi
+        depth_sensor.register_metadata(RS2_FRAME_METADATA_SUB_PRESET_INFO,
```

Comment:
Correct to remove the "added for mipi" comment here - please consider to remove also the other ones in this method.

Reply:
Done